[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/gamayos/finite-ring-space/blob/main/src/38-s13/38-s13-main.ipynb)

## Exact Quantum Dynamics on the Minimal (13,233) Holographic Substrate (Akhtman & Voether, 2026)

**Validation Package.** `finite-ring-space/src/38-s13` — the paper's seven in-tree audits (four node scripts, integer-exact;
`o1_gr_chart.py`, sympy over the declared continuum chart; `check_phi.py`, the golden-ratio audit; `check_fibrations.py`, the two fibrations) and the laboratory's
six node suites (`docs/38-s13/verify-*.js`, run from where the laboratory serves them), through one registry: a family
check is one script (`s13.<stem>`) run as its own process, its micro-checks the script's own PASS/FAIL lines together
with the registry's predicates, which pin the stated totals (58, 10, 10, 11, 7, 10, 30; 84, 28, 25, 25, 26, 20 — the
laboratory's 208) and the lines the paper's rows rest on: the quarter roots ħ = 144, h = 89; the apsidal fraction
3/58 = (p−1)/(Ω−1); the per-chronon rate 1/(Ω−1) at (13,233) and (5,233); the face ratio 2 ⟺ 2γ − β = 1; the covering
parity; the winding family. Each family names the row(s) of the paper's predicate ledger it witnesses (the paper's
subsection "Predicate ledger", rows cited as `38:XN`; public copy at `docs/38-s13/38-s13-ledger.html`), and the ledger's
source column cites the family ids in return. Master-ledger rows of the corpus sourced from this paper: `00:C16`–`00:C19`,
`00:C22`, `00:C25`, `00:D13`, `00:D14`, `00:E8`, `00:E9`, `00:Y6`.

**Kinds** — two, recorded per family. `EXACT`: integer arithmetic (node, python int), no float behind any claim.
`SYMBOLIC`: exact rationals in sympy over a declared continuum comparison chart ([approx]).

**Run.** Cell by cell, or *Runtime → Run all*; ≈ 30 s on Colab (node and sympy are present there). The last cell writes
`results.json` and fails loudly if any family fails.

In [1]:
# --- environment: clone the package and the laboratory if this notebook is not already running inside the package (Colab).
import os
if not os.path.exists("s13common.py"):
    if not os.path.exists("finite-ring-space"):
        !git clone --filter=blob:none --sparse https://github.com/gamayos/finite-ring-space.git
        os.chdir("finite-ring-space")
        !git sparse-checkout set src/38-s13 docs/38-s13
        os.chdir("src/38-s13")
    else:
        os.chdir("finite-ring-space/src/38-s13")
print("working directory:", os.path.join(*os.getcwd().split(os.sep)[-3:]))
!node --version

working directory: finite-ring-space/src/38-s13
v22.23.2


In [2]:
import importlib, s13common as gc
importlib.reload(gc)         # a fresh registry if the notebook is re-run
print("families registered:", len(gc.LABELS))

families registered: 13


## The in-tree audit: the pair, the tower, the quarter roots, the flow, the covering
Integer arithmetic on F₁₃ and F₂₃₃: the pair constants and the tower identity C₁₂ = C₄ × C₃ (with its general form 4a − κb ≡ 1 mod 4κ), the channel gcds, the Carrier quarter roots 78⁵⁸ = 89 = h and ħ = 144 (ħh = 1, ħ + h = Ω), the dilation instance 3/58 = (p−1)/(Ω−1) and the per-chronon leak rate (κ/S)/(p−1) = 1/(Ω−1), κ-free at both (13,233) and (5,233) (C10), the (53,13) kill test, the meridian stations, the covering 144 = 36 + 108, the ramification and fusion congruences, the registration fibre product |R| = 696; the golden-ratio audit of the quarter roots (h = 13/8, ħ = 8/13 on F₂₃₃, the Fibonacci convergents).

| id | script | kind | claim | ledger rows |
|---|---|---|---|---|
| `s13.check_s13` | `src/38-s13/check_s13.js` | EXACT | the in-tree audit, 58 integer checks: the pair (13, 233), κ = 3, S = 58, g = 2, i = 5; the tower identity C₁₂ = C₄ × C₃ with its general form 4a − κb ≡ 1 (mod 4κ); the channel gcds and N(i) = −1; the Carrier quarter roots 78⁵⁸ = 89 = h, ħ = 144, ħh = 1, ħ + h = Ω; the algebra half ½ = 7; the frame counts 2184 = 156 × 14; the resolvable window; the dilation instance 3/58 = 12/232 = (p−1)/(Ω−1) and the per-chronon rate (κ/S)/(p−1) = 1/(Ω−1) at (13,233) and (5,233); the (53,13) kill test in Carrier 157; the meridian stations, the covering 144 = 36 + 108, the ramification 13/(2m), the fusion 5·39 ≡ −13 (104); the registration fibre product |R| = 696 | 38:A1, 38:A2, 38:A3, 38:A4, 38:A6, 38:A7, 38:A8, 38:C6, 38:C10, 38:D2, 38:D6, 38:D7, 38:D8, 38:D9, 38:V3 |
| `s13.check_phi` | `src/38-s13/check_phi.py` | EXACT | the golden-ratio audit of the quarter roots (A4), 10 integer checks: h = 13/8 ≡ 89 and ħ = 8/13 ≡ 144 on F₂₃₃ (the Fibonacci convergents F₇/F₆), φ present iff 5 is a square (absent at 233, present at 30089), the Pisano closure in-register or in the quadratic extension, the Y6 lock constraint κ_O = 3N ⇒ q = 12N + 1 prime (registrable N = 1, 3, 5, 6, 8, 9) | 38:A4 |

Ledger rows witnessed: 38:A1, 38:A2, 38:A3, 38:A4, 38:A6, 38:A7, 38:A8, 38:C6, 38:C10, 38:D2, 38:D6, 38:D7, 38:D8, 38:D9, 38:V3.

In [3]:
for fam in ["s13.check_s13", "s13.check_phi"]:
    print(f"\n— {gc.FILE[fam]}"); gc.run_block(fam)


— src/38-s13/check_s13.js
PASS  p = 13 = 4k+1, k=3
PASS  Om = 233 = 4S+1, S=58
PASS  g = 2 generates F13x (order 12)
PASS  i = 5, i^2 = -1 on F13
PASS  i = g^9 = g^{-3} (heights mod 12)
PASS  g^3 = 8, -g^3 = 5 = i
PASS  scale step g^4 = 3, order kappa = 3
PASS  quarter Ihat = g^{-kappa} = 5, order 4
PASS  THE TOWER IDENTITY g = g^4 * g^{-3} = 3*5 = 2
PASS  4 kappa = p-1 = 12 (tower closes in one drive revolution)
PASS  C12 = C4 x C3 (gcd(4,3)=1)
PASS  1-algebra half (p+1)/2 = 7; -1/2 = 6
PASS  gcd(p-1,p+1) = gcd(12,14) = 2
PASS  gcd(p-1,2(p+1)) = gcd(12,28) = 4 (Q4)
PASS  4 gcd(kappa,2kappa+1) = 4 (k=3: gcd(3,7)=1)
PASS  78^58 = 89 on F233 (the quarter root h)
PASS  h = 89: h^2 = -1 on F233
PASS  hbar = 144: hbar^2 = -1 on F233
PASS  hbar * h = 144*89 = 1 on F233 (inverse pair)
PASS  hbar + h = 144 + 89 = 233 = Om
PASS  Fibonacci: F13 = 233, F12 = 144, F11 = 89, F7 = 13
PASS  233 = 13^2 + 8^2
PASS  |PGL2(13)| = 13*12*14 = 2184 = 156*14
PASS  cone chart cells p(p-1) = 156; fibres of p+

P1 FORCED: the two roots of -1 are x and Om-x, so hbar + h = Om at every admissible Carrier. 144+89=233 is a definition, not a Fibonacci fact.
P2 FORCED: admissibility requires Om == 1 (mod 4), so every Carrier has a unique two-square form. 233 = 13^2+8^2, 857 = 29^2+4^2, 30089 = 160^2+67^2.
P3 THEOREM: for Om = a^2+b^2, (a/b)^2 == -1. So {hbar,h} ARE the two Gaussian leg ratios. At (13,233): h = 13/8 = 89, hbar = 8/13 = 144 (the paper's A4). Holds at all four.
P3a REGULARITY, NOT LAW: p is the larger leg at (13,233) and (29,857) (Om - p^2 a perfect square: 64, 16) and NOT at (173,30089) (30089 - 173^2 = 160, no square). Record with O5's caution.
P4 RECURS ONCE, OUT OF REACH: 233 = F_13, and F_(2n+1) = F_n^2+F_(n+1)^2 forces consecutive-Fibonacci legs, hence h = F_7/F_6 = 13/8, a phi convergent (hbar = 8/13). Among ALL Fibonacci primes through F_89 exactly three satisfy Om == 41 (mod 48): F_11=89 (too small -- p^2 < 89 leaves only the degenerate p=5), F_13=233, and F_83=991948530947554

## The gravitational face: the two faces of the leak and the comparison chart
The two faces of the leak, angular κ/S and temporal κ/(2S), ratio exactly 2 (the double cover, C7); the face-ratio identity 2 ⟺ 2γ − β = 1 over the PPN lattice (C8); the dictionary p_sl = 3(S/κ) r_g and its capacity reading — the dictionary C12, a definition — in exact rationals; the continuum comparison chart in sympy [approx: continuum chart]: the parametrised isotropic metric's apsidal and clock-deficit coefficients, the ratio deviation (2/3)(2γ − β − 1), Brans–Dicke's −2/(2+ω) with its GR limit.

| id | script | kind | claim | ledger rows |
|---|---|---|---|---|
| `s13.check_o1` | `src/38-s13/check_o1.js` | EXACT | the dictionary audit, 10 exact-rational checks: the two faces of the leak (angular κ/S, temporal κ/(2S)), ratio exactly 2, the double cover; the closure structure 696/1392 and the half event; the dictionary p_sl = 3(S/κ) r_g and its capacity reading p_sl = S r_g at κ = 3; the p = 53 regression (modulus 157); face ratio 2 ⟺ 2γ − β = 1 over the half-integer PPN lattice | 38:C7, 38:C8, 38:C12, 38:V4 |
| `s13.o1_gr_chart` | `src/38-s13/o1_gr_chart.py` | SYMBOLIC | the continuum comparison chart for the face-ratio identity (C8) and the dictionary (C12), 10 sympy identities [approx: continuum chart]: the parametrised isotropic metric's apsidal coefficient 2 − β + 2γ and clock-deficit coefficient 3/2 (PPN-free), the ratio deviation (2/3)(2γ − β − 1), the dictionary match p_sl = 3(S/κ) r_g, the angular face 3r_g/p_sl = κ/S automatically; Brans–Dicke's combination −2/(2+ω) with the ω → ∞ limit | 38:C8, 38:V4 |

Ledger rows witnessed: 38:C7, 38:C8, 38:C12, 38:V4.

In [4]:
for fam in ["s13.check_o1", "s13.o1_gr_chart"]:
    print(f"\n— {gc.FILE[fam]}"); gc.run_block(fam)


— src/38-s13/check_o1.js
pass (a) angular face (the curl): one Carrier tick per chronon -> (p-1)/(Om-1) = 12/232 = 3/58 = kappa/S per revolution -- the apsidal fraction (C18)
pass (a) temporal face (the ring): the halved tick per chronon -> 12/464 = 3/116 = kappa/(2S) per revolution (B7, the sheet on C464)
pass (a) face ratio exactly 2: the double cover
pass (b) joint cycle L = lcm(12,232) = 696 = S*12 = kappa*232; cover 2L = 1392
pass (b) half event at L/2 = 348: shell home (348 = 0 mod 12), Carrier at -1 (348 = 116 mod 232, 78^116 = -1 on F233): the sign face, base-transportable (Thm 6)
pass (c) matching temporal face to deficit: (3/2)(r_g/p) = kappa/(2S) gives r_g/p = kappa/(3S), i.e. p_sl/r_g = 3S/kappa = 58 at (13,233)
pass (c) capacity reading at kappa = 3: p_sl = S r_g -- one gravitational radius per decoherence step, the D3 reading; separation count = the capacity
pass (c) consistency: the angular face then equals 3 r_g/p_sl = 3/(3S/kappa) = kappa/S -- the GR apsidal fraction,

pass (1) exponential metric: apsidal fraction coefficient = 3 exactly
pass (1) exponential metric: clock-deficit coefficient = 3/2 exactly
pass (1) face ratio = 2 exactly
pass (2) PPN apsidal coefficient = 2 - beta + 2*gamma
pass (2) PPN deficit coefficient = 3/2, beta/gamma-free
pass (2) ratio = 2  <=>  2*gamma - beta = 1  (the ratio deviates by exactly (2/3)(2*gamma - beta - 1))
pass (3) matching the temporal face: p_sl = 3 (S/kappa) r_g
pass (3) the angular face then reads 3 r_g/p_sl = kappa/S: the apsidal fraction, automatically
pass (4) Brans--Dicke combination 2*gamma - beta - 1 = -2/(2+omega), nonzero at every finite omega (deviation -4/(3(2+omega)))
pass (4) the omega -> oo limit returns the forced value (general relativity)
ALL O1 GR-CHART IDENTITIES PASS
PASS 10 verdict lines, 0 failing, exit 0: the stated 10 checks all pass
PASS [approx: continuum chart] the ratio deviation (2/3)(2 gamma - beta - 1); the dictionary match p_sl = 3 (S/kappa) r_g; Brans--Dicke's -2/(2+omega) wi

## The two fibrations of the frame variety
The frame variety of order p(p²−1) carries two torus fibrations with one count (A9, master C22): the split fibration by the drive torus C_{p−1} over the unit 2-sphere, p(p+1) points, and the non-split fibration by the boost torus C_{p+1} over the nonsquare-radius 2-sphere, p(p−1) points — the frame triple G ≅ P¹ × F_p × C_{p−1} and the boundary clause, exact at p = 5, 13, 17 (the laboratory's `verify-hopf` carries the same content at p = 13 and p = 5).

| id | script | kind | claim | ledger rows |
|---|---|---|---|---|
| `s13.check_fibrations` | `src/38-s13/check_fibrations.py` | EXACT | the two torus fibrations of the frame variety (A9, master C22), 30 exact checks at p = 5, 13, 17: the split fibration by the drive torus C_{p−1} over the unit 2-sphere (p(p+1) points) and the non-split fibration by the boost torus C_{p+1} over the nonsquare-radius 2-sphere (p(p−1) points), the same count p(p²−1); the frame triple G ≅ P¹ × F_p × C_{p−1}; the boundary clause | 38:A9, 38:X4 |

Ledger rows witnessed: 38:A9, 38:X4.

In [5]:
for fam in ["s13.check_fibrations"]:
    print(f"\n— {gc.FILE[fam]}"); gc.run_block(fam)


— src/38-s13/check_fibrations.py



=== p = 5, nu = 2, eta(-1) = 1 ===
PASS p=5: |S^1| = #{x^2+y^2=1} = p - eta(-1) (p-1 at p=4k+1: the drive torus; p+1 at p=4k+3)  [4, 6]
PASS p=5: the circle subgroup is the diagonal torus of SL_2, order p-1, inside the Borel  [4, 4]
PASS p=5: SPLIT fibration of S3=SL_2: q -> q i q^-1, fibres = the p(p+1) cosets of S^1, base = the unit 2-sphere {a^2+bc=1} (p(p+1) points)  [30, 30]
PASS p=5: SPLIT: the Borel meets every fibre in the whole fibre or not at all: no section in B (B_SL ∩ S^1 = S^1)
PASS p=5: 40-hyper B2: |S3| = |S^1| |S^2| = (p-1) p(p+1)  [120, 120]
PASS p=5: NON-SPLIT fibration of PGL_2: g -> g u_n g^-1 (u_n^2 = nu, nonsquare), fibres = the p(p-1) cosets of T = C_(p+1), base the nonsquare sphere {a^2+bc=nu} (p(p-1) points)  [20, 20]
PASS p=5: NON-SPLIT: the Borel meets every fibre exactly once (C19's section); B ∩ T = 1
PASS p=5: NON-SPLIT on S3=SL_2: p(p-1) cosets of C_(p+1); the Borel meets a coset in 0 or 2 elements (the sign): no section  [20, [0, 2]]
PASS p=5: involuti

## The covering parity and the winding family
The covering parity theorem (D12): the reflection identity at p = 5, 13, 17, 29, the multiplicities μ ∈ {1, 3} by the row-parity decider, the 72/72 direct/echo balance, Ω-blind at p = 5, 17, 29; the winding family (D13): the winding-1/5 identity, the 144-slot containment, the covering multiset, the axis passages 13j/m (drawn iff m ≥ 4), the ramification 13/(2m).

| id | script | kind | claim | ledger rows |
|---|---|---|---|---|
| `s13.check_o2` | `src/38-s13/check_o2.js` | EXACT | the multiplicity audit, 11 exact checks: the covering parity theorem (D12) — the reflection identity at p = 5, 13, 17, 29, μ ∈ {1, 3} by the row-parity decider, the shell-parity position law, the 72/72 direct/echo balance, the per-node candidate identities, the shell-1 closed form; the decider Ω-blind at p = 5, 17, 29 | 38:D12, 38:V5 |
| `s13.check_o3` | `src/38-s13/check_o3.js` | EXACT | the family audit, 7 exact checks: the winding-1/5 identity with the M₀/M₃ station tables, the 144-slot containment in the drawn observable sector, the covering multiset with parity multiplicities, the axis passages 13j/m (drawn iff m ≥ 4), the ramification 13/(2m) in the observable sector | 38:D13, 38:V6 |

Ledger rows witnessed: 38:D12, 38:D13, 38:V5, 38:V6.

In [6]:
for fam in ["s13.check_o2", "s13.check_o3"]:
    print(f"\n— {gc.FILE[fam]}"); gc.run_block(fam)


— src/38-s13/check_o2.js
pass (a) reflection identity floor(ab/p) + floor((p-a)b/p) = b-1, all a,b, p in {5,13,17,29}
pass (b) 72/72 nodes, 144 slots, mu in {1,3}
pass (b) decider: mu = 3 iff the row is even; 36 nodes once + 36 thrice = 144 = 36 + 108
pass (b) the position law (shell-parity duality): every odd-row node has one slot -- direct (t <= kappa) on the odd shells a = 1, 3, echo on the even shell a = 2; every even-row node has three with the dual split (1 direct + 2 echo on odd shells, 2 direct + 1 echo on the even shell); direct and echo slots balance 72/72
pass (c) per-node candidate identities: A+B = a-1, C+D = p-1-a, A+C = M-1, B+D = p-1-M, four-sum = p-2 (odd) -- all shells, all rows
pass (c) the four-candidate parity count reproduces mu = 3 iff k even, all shells, all rows
pass (d) shell-1 decider in closed form: w(p-k, p-1) = p-1-k, so the deciding sheet parity equals the row parity (p-1 even), every p
pass (e) Omega-blind: mu in {1,3} with the row-parity decider at p =

## The laboratory's six suites
The suites the laboratory at `finitering.space/38-s13/` ships, run from `docs/38-s13` (one source): the pair dynamics, events, octant bridge, precession and the mass–energy channel (84); the sky map, fibered covers, radial ladder and the central product isomorphism (28); the register, cone arithmetic, the winding spectrum and the curl algebra (25); the shell operator core H⁷ = 0, ord(U) = 13, U unitary (25); the two fibrations of the frame variety, Ω-blind at p = 5 (26); the production rendering itself (20) — 208 checks.

| id | script | kind | claim | ledger rows |
|---|---|---|---|---|
| `s13.verify-233` | `docs/38-s13/verify-233.js` | EXACT | the laboratory's pair suite, 84 checks: the minimal pair under the complete admissibility predicate (Ω ≡ 41 mod 48, no earlier complete Carrier), the pair dynamics and events, the octant bridge, the precession 3/58 with the joint cycle 696 and the half event at 348, the mass–energy channel | 38:B5, 38:V1 |
| `s13.verify-sky` | `docs/38-s13/verify-sky.js` | EXACT | the laboratory's sky suite, 28 checks: the node field 72 = 3 × 2 × 12 with retarded labels, the fibered covers, the radial ladder R sin(2aπ/13) with capacity 4a < 13, the mounting ×3 = g⁴, the central product isomorphism φ(a, b) = 13a + 12b mod 312 | 38:B1, 38:B2, 38:B3, 38:V1 |
| `s13.verify-space` | `docs/38-s13/verify-space.js` | EXACT | the laboratory's register suite, 25 checks: the register and cone arithmetic, the winding spectrum H_wind, the curl algebra δ_k = 78^k with δ₅₈ = h, the Schrödinger dictionary ψ_{τ+1} = g ψ_τ | 38:D11, 38:V1 |
| `s13.verify-f13` | `docs/38-s13/verify-f13.js` | EXACT | the laboratory's shell-operator suite, 25 checks: H⁷ = 0, ord(U) = 13, U exactly unitary on F₁₃ | 38:D11, 38:V1 |
| `s13.verify-hopf` | `docs/38-s13/verify-hopf.js` | EXACT | the laboratory's Hopf suite, 26 checks: the non-split fibration by the boost torus with the frame counts 2184 = 156 × 14, SL₂ obstructing at −1; the norm-one sphere ↔ SL₂ (norm = det), the involution counts, the adjoint action PGL₂ → SO₃, the two 2-sphere counts, the centralisers and orbits of the boost axis and of the unit i, the split circle inside the Borel, the bijection G ≅ P¹ × B; Ω-blind at p = 5 | 38:A7, 38:A9, 38:X4, 38:V1 |
| `s13.verify-render` | `docs/38-s13/verify-render.js` | EXACT | the laboratory's rendering suite, 20 checks on the production rendering itself: the sky chronon τ mod 24, the Hopf representation node-on-fiber, the frame leak at the halved tick π/232, the one retarded flow law φ(k) = −(b_C − k)π/116, ownership, the half-turn and the C₄ stations, the capacity stations, the scale-tower circuit, the four rays, the quadrature pair | 38:B4, 38:B6, 38:B7, 38:C1, 38:C2, 38:C3, 38:C4, 38:C5, 38:D1, 38:D3, 38:D4, 38:D5, 38:D10, 38:V1, 38:V2 |

Ledger rows witnessed: 38:A7, 38:A9, 38:B1, 38:B2, 38:B3, 38:B4, 38:B5, 38:B6, 38:B7, 38:C1, 38:C2, 38:C3, 38:C4, 38:C5, 38:D1, 38:D3, 38:D4, 38:D5, 38:D10, 38:D11, 38:V1, 38:V2, 38:X4.

In [7]:
for fam in ["s13.verify-233", "s13.verify-sky", "s13.verify-space", "s13.verify-f13", "s13.verify-hopf", "s13.verify-render"]:
    print(f"\n— {gc.FILE[fam]}"); gc.run_block(fam)


— docs/38-s13/verify-233.js
PASS Subject minimality: kappa=1 gives p=5, its own Q4 core (4k=4, p-1=4); kappa=2 gives 9, composite; kappa=3 gives 13, prime
PASS Omega = 233 prime, = 4S+1, S = 58
PASS complete admissibility: S = 2 (mod 4), S = 1 (mod 3), Omega = 5 (mod 12), p^2 < Omega
PASS jointly: Omega = 41 (mod 48)
PASS the scan of (169, 233) leaves nothing: 173,181,197,229 fail S even; 193 fails S=1(3)
PASS octant exists: Omega = 1 (mod 8)
PASS the completion needs S = 2 (mod 4): the guard Omega = 257 (S = 64) passes S even, S = 1 (mod 3), Omega = 5 (mod 12), p^2 < Omega, yet (257-1)/8 = 32 is even, while (233-1)/8 = 29 is odd
PASS hbar = 2 sqrt(S): 72^2 = 58, hbar = 144, hbar^2 = -1
PASS h = -hbar = 89, G = 2S = 116
PASS c = 74: c^2 = 117 = 2S+1 = 2^{-1}
PASS k_B = 124: k_B^2 = Omega-2 = 231
PASS the gauge bit: k_B c = h selects c = 74 (the branch -c = 159 gives k_B(-c) = hbar)
PASS core Q4 = {1, hbar, -1, h} = {1, 144, 232, 89}: hbar^2 = -1, h = -hbar, closed under multiplication

PASS p = 13: the observable frame group PGL2 has (p-1)p(p+1) = 2184 elements, the Borel p(p-1) = 156, the boost torus p+1 = 14
PASS p = 13: the boost torus is fixed-point-free on P1: the boosts change every observer
PASS p = 13: unique factorization: every frame is exactly one (cone-chart event) x (boost); the Borel meets every Hopf fiber exactly once: the cone chart is a global section of the finite Hopf fibration, 156 fibers of 14
PASS p = 13: the fiber coordinate is the horizon circle: t -> t([1:0]) bijects the boost torus with P1, the cells plus the horizon class [1:0]
PASS p = 5: the observable frame group PGL2 has (p-1)p(p+1) = 120 elements, the Borel p(p-1) = 20, the boost torus p+1 = 6
PASS p = 5: the boost torus is fixed-point-free on P1: the boosts change every observer
PASS p = 5: unique factorization: every frame is exactly one (cone-chart event) x (boost); the Borel meets every Hopf fiber exactly once: the cone chart is a global section of the finite Hopf fibration, 20 fib

PASS the full 696-state pair sweep renders all three panels, both views, both sky representations
PASS the canvases carry no caption text: geometry and data labels only
PASS the outer fiber's twenty-four labels match the exact retarded values
PASS the double cover on the rendered sky (Carrier view, fixed Carrier state): positions close at t24 + 24 and stand antipodal at t24 + 12
PASS the Subject view holds the section and shows the relational observable: the sky is identical across every drive phase and Carrier tick (the rotation and the common flow are gauge), while the retarded boost-curl displaces every image along its leaf by its lookback (00:C18) -- static, integer-exact, visible
PASS the wrap: the Carrier sky steps continuously through tau 695 -> 696 (a single-step size under 80 px, the sheet carrying the half-angle branch) and the Subject sky stands exactly (the curl is lookback-anchored, not clock-driven)
PASS all three panels draw substantively in the Carrier view
PASS the Hop

## Summary
Writes `results.json` (one record per family: id, the ledger row(s) witnessed, script and path, kind, claim, PASS/FAIL, the check counts) and raises if any family failed.

In [8]:
ok = gc.summary(write=True)
failed = [r["id"] for r in gc.RESULTS if not r["ok"]]
assert ok, f"FAILED families: {failed}"
print(f"all {len(gc.RESULTS)} families pass ({len(gc.MICRO)} micro-checks); records in results.json")


SUMMARY: 13/13 checks passed (354 micro-checks)
all 13 families pass (354 micro-checks); records in results.json
